build.gradle

```diff
@@ -18,10 +18,10 @@ repositories {
 }
 
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib"
-    compile "org.jetbrains.kotlinx:kotlinx-coroutines-core:0.17"
-    compile "com.squareup.okhttp3:okhttp:3.8.0"
-    testCompile 'junit:junit:4.12'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:1.2.71"
```

Comment:
Please return `compile`. Those dependencies are public and transitive and required for librayry, they actually `api`, not `implementation`, but `api` requires java-plugin, but in this case it's not so neccessary.
You can keep it for `testCompile`

Reply:
Oops, you're right, sorry for that.